sdk/sphere/Azure.ResourceManager.Sphere/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("Sphere")]
```

Comment:
The value here is `Microsoft.AzureSphere`

Reply:
fixed in latest commit